client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -25,13 +25,12 @@ export const DateFilter: FC<{ filterDefinition: DateFilterDefinition }> = ({
     minDate = '0000-01-01',
   } = filterDefinition;
   const { urlQuery, updateQuery } = useUrlQuery();
-  const { customDate } = useFormatDateTime();
+  const { customDate, urlQueryDate, urlQueryDateTime } = useFormatDateTime();
```

Comment:
I was thinking to add methods for `urlQueryDate` and `urlQueryDateTime` which you could use instead of `customDate`.. but this is fine 🙏